### DISCUSSION TITLE
Install Publii And Sites In Different Folders

### DISCUSSION THREAD

**Discussion by varanasib** [GetPublii/Publii#1762] [Help]
Is there a place to change the directories Publii uses?  I can't find where the config information is set or saved.

Currently, everything is in ~/Documents/Publii  I'd like to separate the app files from the site files.  I especially want the app out of my ~/Documents directory, and I'd really like the app files out of my home directory entirely.

[Edit]
I see this is the subject of this issue: https://github.com/GetPublii/Publii/issues/1212 in May 2023.

**Comment by michaelfluegge** [comment]
and this
https://getpublii.com/docs/install-publii.html#changingwherepubliistoresitswebsitefiles
does not work for you?

**Reply by varanasib** [reply]
Thanks for the answer.  I noticed that setting.  

Unfortunately, while it moves the site files, it doesn't move the app's user files.  Is there another setting to move the app files?

**Reply by michaelfluegge** [reply]
Not as far as I know. But maybe @dziudek can add some news on this
